services/streams/streams/parsers.go

```diff
@@ -61,6 +65,17 @@ func (streamComp *StreamComponent) ToMap() map[string]any {
 	return streamCompMap
 }
 
+func (repetitive *Repetitive) ToMap() map[string]any {
+	repetitiveMap := make(map[string]any)
+	if repetitive.Value != nil {
```

Comment:
```suggestion
	if repetitive!= nil && repetitive.Value != 0 {
```
the default value of an int is 0 and not nil

Reply:
fixed